Core/clim-core/presentation-defs.lisp

```diff
@@ -1304,11 +820,6 @@ protocol retrieving gestures from a provided string."))
   (declare (ignore acceptably for-context-type))
   (prin1 object stream))
 
-(defmethod presentation-type-of ((object symbol))
-  (if (eq (symbol-package object) (find-package :keyword))
-      'keyword
-      'symbol))
```

Comment:
Why are these moved? I think it makes sense to defined standard presentation types along with "their" methods.

Reply:
presentation-type-of is specified as a function in this part of the spec. moreover, it doesn't return the presentation-type, only the symbol. I think that it is justified to put it there.